docs/clusterinterceptors.md

```diff
@@ -62,13 +49,12 @@ spec:
       port: 8081 # defaults to 80
 ```
 
-## Interceptor Services
-
-To be a valid ClusterInterceptor, the workload should satisfy the following:
+## Configuring a Kubernetes Service for the `ClusterInterceptor`
 
-- Be fronted by a regular Kubernetes v1 Service and serve HTTP
-- Accept HTTP POST requests that contain a [`InterceptorRequest`](https://pkg.go.dev/github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1#InterceptorRequest) 
-  JSON body
-- Respond with a status code of 200 OK with [`InterceptorResponse`](https://pkg.go.dev/github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1#InterceptorResponse) 
-  body
-- Respond with a non 200 response only if something catastrophic went wrong during processing
+The Kubernetes object running the custom business logic for your `ClusterInterceptor` must meet the following criteria:
+- Fronted by a regular Kubernetes v1 Service listening on an HTTP port (default port is 80)
+- Accepts an HTTP `POST` request that contains an [`InterceptorRequest`](https://pkg.go.dev/github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1#InterceptorRequest) 
+  as a JSON body
+- Returns an HTTP 200 OK response that contains an [`InterceptorResponse`](https://pkg.go.dev/github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1#InterceptorResponse) 
+  as a JSON body
+- Returns a response other than HTTP 200 OK only if payload processing halts due to a catastrophic failure
```

Comment:
Should this explicitly mention using `continue: false` instead of an error when the interceptor should filter Trigger processing?

Reply:
That's a question for @dibyom

Reply:
Yes, it should. Will fix.

Reply:
I created https://github.com/tektoncd/triggers/issues/1056 to keep track of this. Let's fix this in a separate PR.